CHANGES.rst

```diff
@@ -36,7 +36,9 @@ Improvements
 Bugfixes
 ^^^^^^^^
 
-- None so far :)
```

Comment:
this needs a rebase, the changelog in master already contains a bugfix

Reply:
Done